src/lib/gui/config/AppConfig.h

```diff
@@ -325,6 +329,7 @@ class AppConfig : public QObject, public deskflow::gui::IAppConfig
   std::optional<bool> m_EnableUpdateCheck;
   bool m_logExpanded = true;
   bool m_colorfulTrayIcon = false;
+  bool m_RequireClientCert = false;
```

Comment:
Less secure by default is less secure. Would defaulting the 'require client certs' setting to `true` make sense? I fear with this disabled, 95% of users won't ever turn it on.

```suggestion
  bool m_RequireClientCert = true;
```